spec/services/reports/region_cache_warmer_spec.rb

```diff
@@ -38,10 +38,9 @@ def refresh_views
 
   it "warms the cache for all regions" do
     _facilities = FactoryBot.create_list(:facility, 5, block: "Block 1", facility_group: facility_group_1)
-
-    expect(Reports::RegionService).to receive(:call).with(hash_including(region: instance_of(Region))).exactly(4).times
-    expect(Reports::RegionService).to receive(:call).with(hash_including(region: instance_of(FacilityGroup))).exactly(2).times
-    expect(Reports::RegionService).to receive(:call).with(hash_including(region: instance_of(Facility))).exactly(10).times
+    # we don't cache the Root or Organization regions
+    regions_to_cache = Region.count - 2
```

Comment:
This assertion is a bit hand-wavy 😅  I agree, it's awkward to assert specific regions. Could we instead add a `expect().not_to receive().with(Org/Root region)`? That will confirm that the 2 missing cache calls are for the root and org regions.